.azure-pipelines/client.yml

```diff
@@ -33,6 +33,20 @@ jobs:
   - task: PublishBuildArtifacts@1
     displayName: 'Publish'
 
+- job: 'Audit'
+
+  dependsOn:
+  - 'Build'
+  
+  pool:
+    vmImage: 'vs2017-win2016'
+
+  steps:
+    - task: ms.vss-governance-buildtask.governance-build-task-component-detection.ComponentGovernanceComponentDetection@0
+      #condition: and(succeeded(), ne(variables['Build.Reason'],'PullRequest'))
```

Comment:
I assume you plan to add this condition back and also change it to match `ne(variables['System.TeamProject'], 'public')`, that we wit will be disabled for public but will run for other others.

Reply:
Yes added